TRENZ.Lib.RazorMail.Core/Models/MailHeaderCollection.cs

```diff
@@ -185,7 +185,7 @@ public void AddBlindCarbonCopy(IEnumerable<MailAddress> addresses) =>
     /// Gets the headers that are not mail addresses and therefore have no getter/setter.
     /// </summary>
     [Obsolete("This will be removed in a future version.")]
-    public IReadOnlyDictionary<string, object> NonAddressHeaders => this
+    private IReadOnlyDictionary<string, object> NonAddressHeaders => this
```

Comment:
Well... if we change the API, it is a breaking change. I'd have favoured a minor release (since it is already marked obsolete). If we make this change, we can outright just remove the property.